ramlfications/parser/types.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015 Spotify AB
```

Comment:
I'm not a big fan of putting my name in source code. This file will obviously be modified by lot of other contributors after me, as ramlfication is meant for great success.
In buildbot, we usually put "Copyright The Buildbot Developers"

Would you agree to put "Copyright The Ramlfications developers"